src/index.js

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import cookieParser from "cookie-parser";
+import cors from "cors";
+import dotenv from "dotenv";
+import Models from "./server/models";
+
+dotenv.config();
+
+const app = express();
+app.use(cookieParser());
+app.use(express.urlencoded({extended: true}));
+
+app.use(cors("*"));
```

Comment:
this is not correct.
remove the argument you passing in the cors

Reply:
Done Sir